tests/apps/courses/test_templates_course_detail.py

```diff
@@ -41,62 +45,66 @@ def test_course_cms_published_content(self):
         # The page should not be visible before it is published
         url = page.get_absolute_url()
         response = self.client.get(url)
-        self.assertEqual(response.status_code, 404)
 
         # Publish and ensure content is correct
         page.publish("en")
         response = self.client.get(url)
-        self.assertContains(
-            response,
+        html = str(response.content, encoding="utf8")
+
+        self.assertEqual(response.status_code, 200)
+
+        self.assertInHTML(
```

Comment:
I still feel that assertContains(..., html=True) is better than assertInHtml because it can be passed a response.